src/sentry/options/manager.py

```diff
@@ -39,17 +45,23 @@ class OptionsManager(object):
     Overall this is a very loose consistency model which is designed to give simple
     dynamic configuration with maximum uptime, where defaults are always taken from
     constants in the global configuration.
-
-    - Values must be strings.
-    - Empty values are identical to null values which are represented by ''.
     """
     cache = default_cache
-
     logger = logging.getLogger('sentry')
+    registry = None
 
     # we generally want to always persist
     ttl = None
```

Comment:
This is scary, since an error in `set` could result in an old value being persisted indefinitely.